index.js

```diff
@@ -182,6 +197,17 @@ Reverb.prototype.onGatewayCommand = function(command) {
             id: body.key.entryId
           , user: body.key.registeredUserId
         });
+    } else if (command.command == 'PUSH_NOTIFICATION_CHANGE') {
+        var body = JSON.parse(command.payload);
+        if (body.notificationId == "t1") {
```

Comment:
Maybe. There's more assumptions of a single timer elsewhere. If they do have multiple timers later, a better way might be to have an onNotification, where the notification is resolved and the Type attribute is then used to figure out if an onTimer or onAlarm needs to happen next. I feel like this is shippable currently.